packages/api/tests/endpoints/providers/list-providers.js

```diff
@@ -83,6 +78,9 @@ test('CUMULUS-912 GET without pathParameters and with an invalid access token re
 test.todo('CUMULUS-912 GET without pathParameters and with an unauthorized user returns an unauthorized response');
 
 test('default returns list of providerModel', async (t) => {
+  t.context.testProvider = fakeProviderFactory();
+  await providerModel.create(t.context.testProvider);
```

Comment:
This test is mocking the ES query operation that the /providers endpoint uses, so we shouldn't need to create the actual item in Dynamo?

Reply:
This already existed in a before each and I just moved it into this test because this is the only test that actually needs it.

Reply:
It seems like this test is only relying on the stubbed output below, so actually creating the provider shouldn't be necessary here either?

Reply:
... in fact, this shouldn't need a stub at all?